spec/core_functions/color/oklab/error.hrx

```diff
@@ -0,0 +1,259 @@
+<===> unit/lightness/input.scss
+a {b: oklab(1px 2 3)}
+
+<===> unit/lightness/error
+Error: $lightness: Expected 1px to have no units or "%".
+  ,
+1 | a {b: oklab(1px 2 3)}
+  |       ^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> unit/a/input.scss
+a {b: oklab(1% 2px 3)}
+
+<===> unit/a/error
+Error: $a: Expected 2px to have no units or "%".
+  ,
+1 | a {b: oklab(1% 2px 3)}
+  |       ^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> unit/b/input.scss
+a {b: oklab(1% 2 3px)}
+
+<===> unit/b/error
+Error: $b: Expected 3px to have no units or "%".
+  ,
+1 | a {b: oklab(1% 2 3px)}
+  |       ^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> unit/alpha/slash/input.scss
+a {b: oklab(1% 2 3/0.4px)}
+
+<===> unit/alpha/slash/error
+Error: $alpha: Expected 0.4px to have no units or "%".
+  ,
+1 | a {b: oklab(1% 2 3/0.4px)}
+  |       ^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> unit/alpha/slash_list/input.scss
+@use 'sass:list';
+a {b: oklab(list.slash(1% 2 3, 0.4px))}
+
+<===> unit/alpha/slash_list/error
+Error: $alpha: Expected 0.4px to have no units or "%".
+  ,
+2 | a {b: oklab(list.slash(1% 2 3, 0.4px))}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 2:7  root stylesheet
+
+<===>
+================================================================================
+<===> type/lightness/input.scss
+a {b: oklab(c 2 3)}
+
+<===> type/lightness/error
+Error: $channels: Expected lightness c to be a number.
+  ,
+1 | a {b: oklab(c 2 3)}
+  |       ^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> type/a/input.scss
+a {b: oklab(1% c 3)}
+
+<===> type/a/error
+Error: $channels: Expected a c to be a number.
```

Comment:
Is there a better way to phrase this error? (maybe "Expected a-channel c to be a number"?)